packages/@expo/cli/src/start/platforms/PlatformManager.ts

```diff
@@ -136,7 +156,7 @@ export class PlatformManager<
       resolveSettings.device
     );
     if (options.runtime === 'expo') {
-      return this.openProjectInExpoGoAsync(resolveSettings);
+      return this.openProjectInExpoGoOrInterstitialPageAsync(resolveSettings);
```

Comment:
```suggestion
      return this. openProjectInExpoGoAsync(resolveSettings);
```

I'd prefer this name stay the same for the [reasoning here](https://github.com/expo/expo/pull/18836/files#r966079250).